client/gatsby-ssr.js

```diff
@@ -0,0 +1,25 @@
+
+/* this is a hack to fix `gatsby-plugin-sass` from including the
+  css in every single file (which is > 120k lines).
+  see: https://github.com/gatsbyjs/gatsby/issues/1526
+  `ssr` means server-side rendering.
+  */
+export const onPreRenderHTML = ({getHeadComponents}) => {
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
so if I'm reading this correctly, we only do this on dev builds? Is the intention to speed up local development builds?  Not sure I'm following the larger reasoning

Reply:
Excellent point. It always makes sense to do it. I could see maybe NOT doing it on dev builds?

Reply:
I think this code says if it's not production, then exit. Meaning it only runs on production. This makes sense if dev has sourcemap files issues I guess?

Reply:
oh no, I missed the `!`! ha, good to know

Reply:
Heh. When you said, "why not production?", I looked at the same code and was like "YEAH, Why is this backwards!!1!".. Then I really looked at it a second time and saw it was prod only... so, I think it's written oddly and a least deserves a comment.

Reply:
I totally missed the `not` there